drush/Commands/cgov/CgovUserCommands.php

```diff
@@ -298,6 +298,11 @@ protected function loadAdditionalUser($user) {
     else {
       $pwd = NULL;
     }
+    $preferred_admin_langcode = '';
```

Comment:
This is not needed here. You can do this down at 322. If the yaml has the key, then you need to be setting it on the user object. full stop.